docs/AdvancedContributorGuidelines.md

```diff
@@ -0,0 +1,49 @@
+# Advanced Contributor Guidelines
+
+## Choosing issues
+
+* Follow issue priorities. Prefer to take higher priority issues over lower priority issues.
+* Favor issues related to certain features/aspects/techs to deepen expertise in those areas. 
+ 
+## Creating commits in big PRs
```

Comment:
May wish to define what is meant by "big".

For me, a "big PR" is when the number of logical changes it makes becomes too complex. (And we then need to define what a "logical change" is :-) ) An estimation I make is when the overall diff of a PR starts to approach around 100 lines (both additions and deletions).

Also I think, "breaking commits up for big PRs" is a hard requirement, non-advanced contributors are not exempt from it. So, I'd advice contributors unfamiliar with Git to either 1. Learn how to use Git first, or 2. avoid taking complex issues that require complex PRs to solve.